Users/Tutorial.rst

```diff
@@ -196,6 +196,12 @@ option to directly open the file in an editor of your choice.
 
 .. note::
 
+    If you want to set a default editor and not be asked for one every time,
+    you can simply add `editor=$editorName` (i.e. editor=vim) to your project's
+    .coafile and it will automatically open in that one.
```

Comment:
IMO `.coafile` should use double backtick as well, like the other "`.coafile`" instances in the file

Reply:
Not sure about this one, except that for consistency you are definitely right.
https://stackoverflow.com/questions/22256995/restructuredtext-in-sphinx-and-docstrings-single-vs-double-back-quotes-or-back
https://mail.zope.org/pipermail/grok-dev/2008-December/006823.html
The name `.coafile` is something that could be / should be, an implicit reference to another section of the documentation.
( And I've created https://github.com/coala/projects/issues/391 about this problem. )